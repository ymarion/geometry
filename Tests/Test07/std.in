# Initializing our objects
C C1 0 0 10
R R1 0 0 20 20
L L1 0 0 20 20
PL PL1 0 0 10 20 20 10 0 0
# And after that, agregation !
OA agg1 C1 R1 L1 PL1
# Should show agg1
LIST


