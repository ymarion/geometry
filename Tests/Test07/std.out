OK
C C1 0 0 10
R R1 0 0 20 20
L L1 0 0 20 20
OA agg1 C1 R1 L1 PL1

